egs/ami/s5b/local/chain/multi_condition/tuning/run_tdnn_lstm_1a.sh

```diff
@@ -109,13 +104,13 @@ local/nnet3/multi_condition/run_ivector_common.sh --stage $stage \
 local/nnet3/prepare_lores_feats.sh --stage $stage \
                                    --mic $mic \
                                    --nj $nj \
-                                   --min-seg-len $min_seg_len \
+                                   --min-seg-len "" \
```

Comment:
wouldn't it be cleaner to remove the min-seg-len option from this script entirely?  The run_ivector_common.sh script already doesn't seem to support it.

But I'm concerned that this PR will break the other chain example scripts and the other nnet3 example scripts.